src/test/java/org/folio/rest/jaxrs/model/ServicePointTest.java

```diff
@@ -0,0 +1,61 @@
+package org.folio.rest.jaxrs.model;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.IOException;
+
+import org.folio.rest.tools.utils.ObjectMapperTool;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.vertx.core.json.JsonObject;
+
+public class ServicePointTest {
+
+  @Rule
+  public ExpectedException expectedExceptions = ExpectedException.none();
+
+  @Test
+  public void canDeserializeFromJson() throws IOException {
+
+    final ObjectMapper mapper = ObjectMapperTool.getMapper();
+
+    final JsonObject servicePointJson = new JsonObject();
+
+    servicePointJson.put("name", "Collection Point A");
+    servicePointJson.put("code", "cpA");
```

Comment:
"discoveryDisplayName" is a required property, why is it missing here?

Reply:
It would appear that the deserialization does not perform any checks on required properties.

I can easily add it, though I quite like that this also describes that aspect of this behaviour.

Reply:
I've added a separate test to demonstrate that required properties aren't required for deserialization. And updated the other tests to include the `discoveryDisplayName` property.